server/channels/app/team_test.go

```diff
@@ -635,7 +673,8 @@ func TestPermanentDeleteTeam(t *testing.T) {
 	require.Nil(t, err, "Should create a team")
 
 	defer func() {
-		th.App.PermanentDeleteTeam(th.Context, team)
+		appErr := th.App.PermanentDeleteTeam(th.Context, team)
+		require.NotNil(t, appErr)
 	}()
```

Comment:
I think we should just remove this clean. The test calls `PermanentDeleteTeam` later and then checks that the command was also deleted. Please add back this code block

```
	err = th.App.PermanentDeleteTeam(th.Context, team)
	require.Nil(t, err)

	command, err = th.App.GetCommand(command.Id)
	require.Nil(t, command, "command wasn't deleted")
	require.NotNil(t, err, "should not return an error")
```
and remove the cleanup defer clause